### PR TITLE
completed exercise

### PR DESCRIPTION
**Pull request by t-doan**
javascript-conditions-1
javascript-conditions-2

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  if (pH === 7) {
+    return console.log('neutral');
+  } else if (pH <= 14 && pH > 7) {
+    return console.log('base');
+  } else if (pH >= 0 && pH < 7) {
+    return console.log('acid');
+  } else {
+    return console.log('Invalid pH level');
+  }
+}
```
Comment (shawnkost) [inline javascript-conditionals/main.js:65]: Rather than `return`ing a `console.log`, you should just be `return`ing the string directly

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tai!

File: javascript-conditionals/main.js (hunk 1 above)
Comment (t-doan) [reply javascript-conditionals/main.js]: Fixed!

**Review by t-doan — COMMENTED** [review]
Made the changes

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**